File: smallest.py
a=[]
n=int(input("Enter no of elements:"))
for i in range(1,n+1):
    b=int(input("enter element:"))
    a.append(b)
a.sort()
print(a)
print("largest no:",a[n-1])
print("smallest no:",a[n-2])
print("Second largest no:",a[1])

File: Smallest.java
import java.util.*;
class Smallest
{
public static void main(String args[])
{
int a;
int arr[]={55,10,8,90,43,87,95,25,50,12};
System.out.println("Array:"+Arrays.toString(arr));
int count=arr.length;
for(int i=0;i<count;i++){
for(int j=i+1;j<count;j++){
if(arr[i]>arr[j]){
a=arr[i];
arr[i]=arr[j];
arr[j]=a;
}
}
}

System.out.println("Smallest:"+arr[0]);
System.out.println("largest:"+arr[count-1]);
System.out.println("Second largest:"+arr[count-2]);
}
}